README.md

```diff
@@ -122,8 +122,7 @@ return static function (ContainerConfigurator $configurator): void
 That could be used out of the box in `knp_paginator.template.pagination` key:
 
 * `@KnpPaginator/Pagination/sliding.html.twig` (by default)
-* `@KnpPaginator/Pagination/twitter_bootstrap_v4_pagination.html.twig`
-* `@KnpPaginator/Pagination/twitter_bootstrap_v5_pagination.html.twig`
+* `@KnpPaginator/Pagination/twitter_bootstrap_v4-v5_pagination.html.twig`
```

Comment:
What about other templates named "twitter_bootstrap_v4_something?"

Reply:
as we modified the PR, now it closed against https://github.com/KnpLabs/KnpPaginatorBundle/pull/678